### ISSUE TITLE
Only one parameter is resolved in url with multiple parameter 

### ISSUE THREAD

**Issue by ptournem** [icimslabs/redux-rest-actions#8]
Hi,

Actually, when you configure an url with multiple parameters, only the last parameter is replaced with its value when you call the API function. 

Version : `1.1.4` 
### API CONFIG : 
```
getContentDetail: {
		url: "/entity/:id/content/:content",
		actions: [
			actions.getContentDetail,
			actions.getContentDetailSuccess,
			actions.getContentDetailError,
		],
	},
```

### API CALL : 
```
api.getContentDetail({
    id : 1,
    content : 2,
});
```

### Expected Behaviour : 
url called : `/entity/1/content/2`

### Actual Behaviour : 
url called : `/entity/:id/content/2`

**Comment by billw-icims** [comment]
Sorry for the delay. Preparing an update with some security fixes this week, will include a fix for this.

**Comment by billw-icims** [comment]
This has been fixed in 1.1.5

**Comment by billw-icims** [comment]
Fixed and published in 1.1.5
